data_mappings.bal

```diff
@@ -0,0 +1,437 @@
+        excludedPaths: let var exPaths = <json[]>payload.excludedPaths in convertToExcludedPathsArray(exPaths),
+        automatic: <boolean>payload.automatic
+    };
+}
+
+# Maps the JSON response returned from the request into record type of IncludedPath.
+# 
+# + payload - The JSON object returned from request
+# + return - An instance of record type IncludedPath
+isolated function mapJsonToIncludedPathsType(json payload) returns @tainted IncludedPath {
+    IncludedPath includedPath = {
+        path: let var path = payload.path in path is string ? path : EMPTY_STRING
+    };
+    if (payload.indexes is error) {
+        return includedPath;
+    }
+    includedPath.indexes = let var indexes = <json[]>payload.indexes in convertToIndexArray(indexes);
+    return includedPath;
+}
+
+# Maps the JSON response returned from the request into record type of ExcludedPath.
+# 
+# + payload - The JSON object returned from request
+# + return - An instance of record type ExcludedPath
+isolated function mapJsonToExcludedPathsType(json payload) returns @tainted ExcludedPath {
+    return {
+        path: let var path = payload.path in path is string ? path : EMPTY_STRING
+    };
+}
+
+# Maps the JSON response returned from the request into record type of Document.
+# 
+# + payload - A JSON object returned from request
+# + return - An instance of record type Document
+isolated function mapJsonToDocumentType(json payload) returns @tainted Document {
+    return {
+        id: <string>payload.id,
+        resourceId: <string>payload._rid,
+        selfReference: <string>payload._self,
+        eTag: <string>payload._etag,
+        documentBody: let var body = <map<json>>payload in mapJsonToDocumentBody(body)
+    };
+}
+
+# Format the JSON map returned from the request to contain only the document. 
+# 
+# + reponsePayload - A JSON map which contains JSON payload returned from the request
+# + return - JSON object which contains only the document
+isolated function mapJsonToDocumentBody(map<json> reponsePayload) returns map<json> {
+    final var keysToDelete = [JSON_KEY_ID, JSON_KEY_RESOURCE_ID, JSON_KEY_SELF_REFERENCE, JSON_KEY_ETAG, 
+            JSON_KEY_TIMESTAMP, JSON_KEY_ATTACHMENTS];
+    foreach var keyValue in keysToDelete {
+        _ = reponsePayload.removeIfHasKey(keyValue);
+    }
+    return reponsePayload;
+}
+
+# Maps the JSON response returned from the request into record type of PartitionKey.
+# 
+# + payload - The JSON object returned from request
+# + return - An instance of record type PartitionKey
+isolated function convertJsonToPartitionKeyType(json payload) returns @tainted PartitionKey {
+    return {
+        paths: let var paths = <json[]>payload.paths in convertToStringArray(paths),
+        keyVersion: <int>payload.'version
+    };
+}
+
+# Maps the JSON response returned from the request into record type of PartitionKeyRange.
```

Comment:
```suggestion
# Maps the JSON response returned from the request into record type of `PartitionKeyRange`.
```

Reply:
Check all similar places.

Reply:
Checked all places in data_mapping.bal file and other files